src/components/Shared/Footer/_About.js

```diff
@@ -12,12 +12,8 @@ const FooterAbout = () => {
         choice - smartphones!
       </p>
       <div className="Footer-About-Vendors">
-        <a href="https://itunes.apple.com/us/app/easya/id1478549100">
-          <img src={FooterAppleLogo} alt="Apple" />
-        </a>
-        <a href="https://play.google.com/store/apps/details?id=io.easya.easya">
-          <img src={FooterGoogleLogo} alt="Google" />
-        </a>
+        <img src={FooterAppleLogo} alt="Apple" />
+        <img src={FooterGoogleLogo} alt="Google" />
```

Comment:
Is it displaying correctly without the `<a>` tag?

Reply:
Yes, it is displaying the exact same without a tag.
Keeping the <a href="#"> was showing warnings so I removed it and the result was same.
![image](https://user-images.githubusercontent.com/54480934/119822159-f83c1080-bf10-11eb-9e87-74164b1d6c42.png)

Reply:
Okay thanks. 👍